### PR TITLE
feat: get external client without passing client id in path #NP-46258

### PR DESCRIPTION
**Pull request by torbjokv**

### CODE REVIEW

File: docs/users-and-roles-swagger.yaml [hunk 1]
```diff
@@ -204,6 +204,29 @@ paths:
         responses: {}
         httpMethod: POST
         type: aws_proxy
+    get:
+      summary: Get a stored client and its associated customer
+      security:
+        - CognitoUserPool: [ "https://api.nva.unit.no/scopes/backend", "https://api.nva.unit.no/scopes/frontend", "aws.cognito.signin.user.admin" ]
```
Comment (sondrev) [inline docs/users-and-roles-swagger.yaml:210]: -ExternalUserPool: []

Comment (sondrev) [reply docs/users-and-roles-swagger.yaml:210]: ExternalCognitoUserPool:
      type: apiKey
      name: Authorization
      in: header
      x-amazon-apigateway-authtype: cognito_user_pools
      x-amazon-apigateway-authorizer:
        type: cognito_user_pools
        providerARNs:
          - Ref: ExternalCognitoAuthorizerArn

File: docs/users-and-roles-swagger.yaml [hunk 2]
```diff
@@ -204,6 +204,36 @@ paths:
         responses: {}
         httpMethod: POST
         type: aws_proxy
+  /external-client-userinfo:
```
Comment (sondrev) [inline docs/users-and-roles-swagger.yaml:207]: /external-client/userinfo ?

**Review by sondrev — APPROVED** [review]

File: docs/users-and-roles-swagger.yaml (hunk 2 above)
Comment (torbjokv) [reply docs/users-and-roles-swagger.yaml:207]: yeah, was not sure about this. what you suggest is in conflict with /external-clients/{clientId}  in my head, how about  /clientinfo ?

**Review by truhacevkir — APPROVED** [review]

**Review by ketilaa — APPROVED** [review]

**Pull request merged**